scipy/special/_add_newdocs.py

```diff
@@ -1046,10 +1052,15 @@ def add_newdoc(name, doc):
     betainc : regularized incomplete beta function
     gamma : gamma function
 
+    Notes
+    -----
+    This function wraps the ``ibeta_inv`` routine from the C++ Boost library [2]_.
```

Comment:
```suggestion
    This function wraps the ``ibeta_inv`` routine from the Boost Math C++ library [2]_.
```

and similarly in the other docstrings. Boost is a set of libraries, not a single library, and the particular library used here is [Boost Math](https://github.com/boostorg/math).